src/main/java/it/gov/pagopa/pu/send/model/SendNotification.java

```diff
@@ -0,0 +1,31 @@
+package it.gov.pagopa.pu.send.model;
+
+import it.gov.pagopa.pu.send.enums.NotificationStatus;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.FieldNameConstants;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.Transient;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Document("send_notification")
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@FieldNameConstants
+public class SendNotification {
+
+  @Transient
+  public static final String SEQUENCE_NAME = "send_notification_sequence";
```

Comment:
I would not store here the sequence name...
It's not obvious its usage...
It could interpreted as the sequence use for the @Id...
Just use the string on the Service that is actually using it